### ISSUE TITLE
What you did with ELK stack

### ISSUE THREAD

**Issue by no00b3** [HadyKh/Feature-Selection-in-DDoS-attacks-problems#1]
Hi, what you did with elk stack? did you doing real time detection? Thanls!

**Comment by HadyKh** [comment]
Hi,

Yes exactly. I got the logs through a docker and implemented the detector in real time.

**Comment by no00b3** [comment]
Hi i'm doing my last project on college to detect intrusion on network

I already did the project and try to implement with dataset  ciccseids2018.
I got high accuracy on dataset, but high false alarm on real time data. I
use TCP Dump to capture into the pcap file and use CICflowmeter to extract
into cicflowmeter format.

 I use this

 https://github.com/iPAS/TCPDUMP_and_CICFlowMeter

Can you help me with your approach? Thanks

**Comment by HadyKh** [comment]
Are you sure that your model is not overfitting the training data?
Also how many features used to train the model?
